site/src/template.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Title
+description: Description
+template: standard
+---
+
+# Page title (H1)
+
+The first thing is the general description.
+
+Then the list of some general classes can be listed, if there are not many of them, or css style specific details
+
+_Example_
+
+The following spacing sizes are supported:
+
+- `s`
+- `m`
+- `l`
+- `none`
+
+Otherwise consider using tables to organize information.
+
+_Example. Table 1_
+
+| cssproperty | value                                                   | screensize |
+| ----------- | ------------------------------------------------------- | ---------- |
+| `margin`    | `xxs`, `xs`, `s`, `m`, `l`, `xl`, `xxl`, `xxxl`, `none` | `tablet`   |
+| `padding`   | `xxs`, `xs`, `s`, `m`, `l`, `xl`, `xxl`, `xxxl`, `none` | `tablet`   |
+
+_Example. Table 2_
+
+| Size   | Value         | Margin example | Padding example |
+| ------ | ------------- | -------------- | --------------- |
+| `xxs`  | .5rem / 8px   | .margin-xxs    | .padding-xxs    |
+| `xs`   | 1rem / 16px   | .margin-xs     | .padding-xs     |
+| `s`    | 1.5rem / 24px | .margin-s      | .padding-s      |
+| `m`    | 2rem / 32px   | .margin-m      | .padding-m      |
+| `l`    | 3rem / 48px   | .margin-l      | .padding-l      |
+| `xl`   | 4rem / 64px   | .margin-xl     | .padding-xl     |
+| `xxl`  | 6rem / 96px   | .margin-xxl    | .padding-xxl    |
+| `xxxl` | 8rem / 128px  | .margin-xxxl   | .padding-xxxl   |
+| `None` | 0             | .margin-none   | .padding-none   |
+
+## Usage (H2)
```

Comment:
I don't think we need `(H2)` here. We should all know what ## means by now 👍

Reply:
(same with the below)

Reply:
I wanted to highlight specifically what heading levels we want to use depending on where are we in the file. Of course, we all know the difference between `#`, `##`, `###`. etc but when you land on a page you might now catch it right away.

So here is a question - will that file be public? Cause I thought we're creating it mostly for ourselves 🤔

Reply:
Updated

Reply:
Still seeing a number of `(H*)` throughout the file